### ISSUE TITLE
Loop Grid Border width is not showing properly

### ISSUE THREAD

**Issue by aditya58singh** [elementor/elementor#22436]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The issue described pertains to the creation of a mobile template, specifically a "Loop Items" template. The template appears to have been designed for mobile devices but when a border was added to the template, it was not functioning as expected. Instead of the border being applied only to the mobile version of the template, it was being taken from the desktop version. This means that the border is not responsive to the device size and is not adapting to the mobile format. This can create a problem with the overall appearance and usability of the template, as the border may not fit or may be too large for the mobile device screen. 

### Steps to reproduce

1. Add a 5px border to the desktop version of the Loop Items template.
2. Adjust the border size to 1px in the mobile version of the template.
3. Observe that the border size remains at 5px in the mobile version of the template.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.4.28
	PHP Version: 8.1.18
	PHP Memory Limit: 384M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://www.staging.androidhire.com
	Home URL: https://www.staging.androidhire.com
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 260M
	Max Memory limit: 384M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 5.5
	Admin Email: aditya58singh@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.4 Safari/605.1.15

== Active Plugins ==
	Ads.txt Manager
		Version: 1.4.2
		Author: 10up

	AMP
		Version: 2.4.1
		Author: AMP Project Contributors

	Broken Link Checker
		Version: 2.0.0
		Author: WPMU DEV

	Elementor
		Version: 3.13.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.1
		Author: Elementor.com

	IndexNow
		Version: 1.0.1
		Author: Microsoft Bing

	Instant Indexing
		Version: 1.1.16
		Author: Rank Math

	Internal Link Juicer
		Version: 2.1.2
		Author: Internal Link Juicer

	Rank Math SEO
		Version: 1.0.114
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.35
		Author: Rank Math

	Starter Templates
		Version: 3.2.1
		Author: Brainstorm Force

	TablePress
		Version: 2.1.3
		Author: Tobias Bäthge

	Web Stories
		Version: 1.32.0
		Author: Google

	WP Rocket
		Version: 3.11.4.2
		Author: WP Media

== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active
	Hello Theme Header & Footer: Active
	Editor Top Bar: Active
	Landing Pages: Active
	Nested Elements: Active
	Lazy Load Background Images: Active
	Page Transitions: Active
	Notes: Active
	Loop: Active
	Form Submissions: Active
	Scroll Snap: Active
	Menu: Active

== Log ==
	
Log: showing 20 of 302023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_1_0_slides Start  
2023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_1_0_slides Finished 
2023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Finished 
2023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Start  
2023-05-12 01:55:41 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Finished 
2023-05-12 01:55:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.4',
  'to' => '3.8.1',
)]
2023-05-12 01:55:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.4',
  'to' => '3.8.1',
)]
2023-05-12 02:01:46 [info] elementor-pro::elementor_pro_updater Started 
2023-05-12 02:01:46 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-12 02:01:46 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-12 02:01:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.1',
  'to' => '3.13.0',
)]
2023-05-12 02:01:46 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.8.1',
  'to' => '3.13.0',
)]
2023-05-12 12:45:18 [info] elementor-pro::elementor_pro_updater Started 
2023-05-12 12:45:18 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-12 12:45:18 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-12 12:45:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]
2023-05-12 12:45:18 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.0',
  'to' => '3.13.1',
)]

PHP: showing 9 of 9PHP: 2023-05-12 01:55:53 [warning X 17][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/site-logo.php::33] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-12 01:56:09 [warning X 19][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-12 01:56:11 [warning X 8][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/site-logo.php::33] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/site-logo.php(33): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/dynamic-tags/data-tag.php(44): ElementorPro\Modules\DynamicTags\Tags\Site_Logo -> get_value()
#2: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Data_Tag -> get_content()
#3: Elementor\Core\DynamicTags\Manager -> get_tag_data_content()
#4: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(109): class type call_user_func_array()
',
)]
PHP: 2023-05-12 02:01:55 [warning X 12][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/site-logo.php::32] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/site-logo.php(32): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/dynamic-tags/data-tag.php(44): ElementorPro\Modules\DynamicTags\Tags\Site_Logo -> get_value()
#2: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Data_Tag -> get_content()
#3: Elementor\Core\DynamicTags\Manager -> get_tag_data_content()
#4: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(109): class type call_user_func_array()
',
)]
PHP: 2023-05-12 02:07:26 [notice X 1][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-12 10:03:49 [error X 1][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func()
#3 /home/androidh/domains/staging.androidhire.com/public_html/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /home/androidh/domains/staging.androidhire.com/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#5 /home/androidh/domains/staging.androidhire.com/public_html/wp-includes/plugin.php(517): WP_Hook->do_action()
#6 /home/androidh/domains/staging.androidhire.com/public_html/wp-admin/admin-ajax.php(188): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-12 15:49:58 [warning X 3][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro-2/modules/theme-builder/classes/conditions-manager.php::171] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-12 17:35:32 [warning X 7][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-12 17:36:50 [warning X 5][/home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro-2/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro-2/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro-2/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor-pro-2/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/androidh/domains/staging.androidhire.com/public_html/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]

JS: showing 3 of 3JS: 2023-05-12 10:18:15 [error X 2][https://www.staging.androidhire.com/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:7826] null is not an object (evaluating 't.target.closest(".elementor").dataset') 
JS: 2023-05-12 10:20:19 [error X 1][https://www.staging.androidhire.com/wp-content/plugins/elementor/assets/js/7fada1a61d72da6d46ab.bundle.min.js:2:7779] null is not an object (evaluating 'elementor.documents.currentDocument.id') 
JS: 2023-05-12 12:04:31 [error X 1][https://www.staging.androidhire.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.2:3:596258] null is not an object (evaluating 'elementor.documents.getCurrent().id') 

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
